src/main/resources/application.properties

```
@@ -3,6 +3,7 @@ spring.datasource.username=root
 spring.datasource.password=156354OlEh
 spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
 spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
-spring.jpa.hibernate.ddl-auto=create-drop
 spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=create-drop
```

remove properties: 
spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect

because they are deprecated and may produce errors

change to validate, you have liquibase for auto creating
```suggestion
spring.jpa.hibernate.ddl-auto=validate
```